python/erskafka/ERSPublisher.py

```diff
@@ -0,0 +1,79 @@
+import os
+import socket
+import inspect
+import ers.issue_pb2 as ersissue
+from datetime import datetime
+from kafka import KafkaProducer
+
+class ERSPublisher:
+
+    def __init__(self, config):
+        """Initialize the ERSPublisher with given Kafka configurations."""
+        self.bootstrap = config['bootstrap']
+        self.topic = config.get('topic', 'ers_stream')
+        self.producer = KafkaProducer(
+            bootstrap_servers=self.bootstrap,
+            value_serializer=lambda v: v.SerializeToString(),
+            key_serializer=lambda k: str(k).encode('utf-8')
+        )
+
+    def _generate_context(self):
+        """Private method to generate the context for an issue."""
+        return ersissue.Context(
+            cwd=os.getcwd(),
+            file_name=__file__,
+            function_name=inspect.currentframe().f_back.f_code.co_name,  # getting the caller function name
+            host_name=socket.gethostname(),
+            line_number=inspect.currentframe().f_back.f_lineno,  # getting the caller's line number
+            package_name="erskafka"
+        )
+
+    def create_issue(self, message, name="DefaultIssueName", severity=None):
+        """Create an ERS issue with minimal user input."""
+        current_time = datetime.now().timestamp()
+        context = self._generate_context()
+        issue = ersissue.SimpleIssue(
+            context=context,
+            name=name,
+            message=message,
+            time=current_time
+        )
+        if severity:
+            issue.severity = severity
+        return issue
+
+    def publish_simple_message(self, message):
+        issue = self.create_issue(message)
+        issue_chain = ersissue.IssueChain(
+            final=issue,
+            session=str(issue.time)  # using time as a session identifier
```

Comment:
take it from enviromental variable: os.getenv('DUNEDAQ_PARTITION') but use "Unknown" as a default value.